docs/forms/FORM_STRUCTURE.md

```diff
@@ -1,422 +1,121 @@
-[Forms structure](#form-structure)
-  1. [Container/presenter and reactive forms](#container-presenter)
-      1. [Form creation in container or in presenter?](#form-creation)
-      2. [Data exchange between container and presenter](#data-exchange)
-        1. [Basic case](#data-exchange-basic)
-        2. [Complex case](#data-exchange-complex)
-      3. [Component Creation](#component-creation)
-        1. [Basic case](#basic-case)
-        2. [Adding complexity ](#adding-complexity)
-            1. [Basic structure](#basic-structure)
-            2. [Include Basic validation](#basic-validation)
-                1. [Validators definition](#validators-definition)
-                2. [Apply validators ](#apply-validators)
-                3. [Validators translations](#validators-translation)
-            3. [Include Custom Validations](#custom-validators)
-                1. [Validators definition](#custom-validators-definition)
-                2. [Apply validators ](#custom-apply-validators)
-                3. [Validators translations](#custom-validators-translations)
-
-<a name="form-structure"></a>
 # Forms structure
 
-Angular provides two approaches for writing the forms, [template-driven forms](https://angular.io/guide/forms) and [model-driven or reactive forms](https://angular.io/guide/reactive-forms).
-This documentation will help you with some best practices to be used at the build of Angular reactive forms components in Otter context.
-
-<a name="container-presenter"></a>
-## [Container/presenter](../components/COMPONENT_STRUCTURE.md) and reactive forms
-Container/presenter architecture was put in place to ensure the best re-usability/sharing
-<a name="form-creation"></a>
-### Form creation in container or in presenter?
+This documentation will help you with some best practices to use when building Angular reactive forms components that have a parent/input component structure (such as container/presenter).
```

Comment:
Maybe mention this is in fact only an implementation of the ControlValueAccessor pattern but we chose to guide them with this example

Reply:
(and that we cover as well the errors but I guess this is in another section :) )